mreg_cli/api/models.py

```diff
@@ -656,7 +656,7 @@ def remove_atom(self, atom_name: str) -> bool:
             if atom_name == atom:
                 break
         else:
-            cli_warning(f"Atom {atom_name!r} not a member of {self.name!r}")
+            raise EntityNotFound(f"Atom {atom_name!r} not a member of {self.name!r}")
```

Comment:
Probably need a different exception type than this? `EntityNotFound` maps to 404 in my mind, but I'm not sure what is a good replacement? Struggling to think of a good name.

Reply:
I used `EntityOwnershipMismatch` for this kind of stuff.